networking_nsxv3/plugins/ml2/drivers/nsxv3/agent/constants_nsx.py

```diff
@@ -233,7 +233,7 @@
                 "id": "HTTP",
                 "display_name": "HTTP",
                 "source_groups": ["ANY"],
-                "destination_groups": ["169.254.169.254"],
+                "destination_groups": ["169.254.169.254", "fe80::a9fe:a9fe"],
```

Comment:
If you like constants we have `neutron_lib.constants.METADATA_V6_IP`. There's also one for v4.